xml/art_jeos_quickstart.xml

```diff
@@ -301,8 +301,145 @@
    <package>cloud-init-config-suse</package>, and
    <package>xfsprogs</package>.
   </para>
- </sect2>
+  <sect3 xml:id="sec-cloud-init-intro">
+  <title>cloud-init basics</title>
+  <para>
+  cloud-init applies automatically custom configuration to running instances.
+  The tool features a wide range of configuration modules, and it can execute
+  specific commands in the image during the boot phase.
+  </para>
+  <para>
+  The default configuration for cloud-init is part of the <package>cloud-init-config-suse</package>
+package, and it is located in the <filename>/etc/cloud/cloud.cfg</filename>
+directory.
+  </para>
+ </sect3>
+<sect3 xml:id="sec-cloud-init-config-examples">
+<title>cloud-init configuration examples</title>
+<para>
+By default, cloud-init creates the <literal>sles</literal> user. It acts as the
+default user with sudo rights. For security reasons, the <literal>root</literal>
+and <literal>sles</literal> users does not have passwords. Use the following example to create passwords for both users.
+</para>
+
+<example xml:id="cloud-init-passwd">
+   <title>Add passwords</title>
+<screen>#cloud-config
+chpasswd:
+list: |
+root:$6$SalTsaLt$.cuqsgjNGxHJZSLmVqkI/j9.kWIiUQHqjlRswbHjTw9hIXdPomgey2DHzmp6Ho6wgqIKz3ufbOIKdQCnQHPSA1
+sles:$6$SalTsaLt$.cuqsgjNGxHJZSLmVqkI/j9.kWIiUQHqjlRswbHjTw9hIXdPomgey2DHzmp6Ho6wgqIKz3ufbOIKdQCnQHPSA1
+expire: False</screen>
+</example>
+
+<para>
+To create the SHA-512 hashes used in the example, run the <command>openssl
+passwd -6 -salt SalTsaLt <replaceable>PASSWORD</replaceable></command> command.
```

Comment:
```suggestion
To create the SHA-512 hashes used in the example, run the command:
<command>openssl passwd -6 -salt SalTsaLt <replaceable>PASSWORD</replaceable</command>.
```

Reply:
The reason I am rephrasing this is that I think "the command" here is `openssl` or possibly `openssl password` and the rest is an example.

Should `SalTsaLT` also be marked [replaceable] as that too looks to me like example text? (I am not familiar with resetting OpenSSL passwords, though.)